Frontend/ui-common/src/Engineering/Helpers/JBridge.purs

```diff
@@ -232,7 +232,7 @@ foreign import emitJOSEvent :: Fn3 String String Foreign Unit
 foreign import getLayoutBounds :: Fn1 String LayoutBound
 foreign import horizontalScrollToPos :: EffectFn3 String String Int Unit
 foreign import withinTimeRange :: String -> String -> String -> Boolean
-foreign import getChatMessages :: String -> Array ChatComponent
+foreign import getChatMessages :: LazyCheck -> Array ChatComponent
```

Comment:
Instead of lazy check, can we make this function `Effect (Array ChatComponent)` ?

Reply:
I am using it in component config also